README.md

```diff
@@ -39,4 +36,36 @@ Now visit [`http://localhost:5000/`](http://localhost:5000/) in your web browser
 The `.env` file is used by flask to set environment variables when running `flask run`. This enables things like developement mode (which also enables features like hot reloading when you make a file change).
 * There's also a [SECRET_KEY](https://flask.palletsprojects.com/en/1.1.x/config/#SECRET_KEY) variable which is used to encrypt the flask session cookie.
 
-When running `setup.sh`, the `.env.template` file will be copied to `.env` if the latter does not exist.
\ No newline at end of file
+When running `setup.sh`, the `.env.template` file will be copied to `.env` if the latter does not exist.
+
+### Run on Vagrant as VM
+```bash
+$ vagrant up
+$ vagrant provision
+$ vagrant up --provision
+
+Some usable commands: 
+$ vagrant reload (to restart/stop app)
+$ vagrant ssh
+$ vagrant suspend
+
+```
+
+### Run on Docker commands:
+```bash
+$ docker build --tag rohanraj14/todo-app .
+$ docker run -p 5000:5000 --env-file .env rohanraj14/todo-app
+```
+
+
+### Run on Docker as VM commands:
+#### 1. Run and Build for local dev environment
+```bash
+$ docker run --env-file ./.env -p 5000:5000 --mount type=bind,source=/.,target=/app rohanraj14/todo-app:dev
```

Comment:
If you're mounting the whole directory, you don't need `--env-file ./.env`

Reply:
@hugh-emersion I need to mount whole dir as my app is in .

Reply:
@hugh-emerson Tried removing the `--env-file ./.env` but it fails for me to run application.